google-basemap/index.ts

```diff
@@ -0,0 +1,133 @@
+import './style.css';
+import {GoogleMapsOverlay as DeckOverlay} from '@deck.gl/google-maps';
+import {colorCategories} from '@deck.gl/carto';
+import {VectorTileLayer, vectorQuerySource} from '@deck.gl/carto';
+
+const GOOGLE_MAPS_API_KEY = import.meta.env.VITE_GOOGLE_API_KEY;
+const GOOGLE_MAP_ID = '';
+const GOOGLE_MAPS_API_URL = `https://maps.googleapis.com/maps/api/js?key=${GOOGLE_MAPS_API_KEY}&v=beta&map_ids=${GOOGLE_MAP_ID}`;
+const apiBaseUrl = import.meta.env.VITE_API_BASE_URL;
+const accessToken = import.meta.env.VITE_API_ACCESS_TOKEN;
+const connectionName = import.meta.env.VITE_API_CONNECTION_NAME;
+const cartoConfig = {apiBaseUrl, accessToken, connectionName};
+const SQL_QUERY = `SELECT geom, country_name, continent_name, pop_2015 FROM cartobq.public_account.world_population_2015 WHERE CAST(pop_2015 as INT64) between CAST(@min as INT64) and CAST(@max as INT64) ORDER BY pop_2015 DESC`;
```

Comment:
For the example, it's clearer to have it together with the source definition, instead of a constant in separated places. The SQL is part of the source definition, so better to have it there.

Reply:
I have casted the column pop_2015 in bigquery to int, so you don't need the CAST anymore

Reply:
Done